src/Bridge/Symfony/DependencyInjection/FidryAliceDataFixturesExtension.php

```diff
@@ -44,12 +41,12 @@ public function load(array $configs, ContainerBuilder $container)
         $processedConfiguration = $this->processConfiguration($configuration, $configs);
         $bundles = array_flip($container->getParameter('kernel.bundles'));
 
-        if (false === array_key_exists(self::NELMIO_ALICE_BUNDLE_CLASS, $bundles)) {
+        if (false === array_key_exists(NelmioAliceBundle::class, $bundles)) {
```

Comment:
> Not sure to understand, if Foo doesn't exist using Foo::class will throw an error.

Nope. PHP converts to string every use of `::class` constants at compile time as documented here: http://php.net/manual/en/language.oop5.constants.php.
So classes don't need to be loaded nor exist before using the constant.
See here for an example: https://3v4l.org/sHMTh